hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAbfsClient.java

```diff
@@ -43,6 +43,8 @@ public ITestAbfsClient() throws Exception {
     super();
   }
 
+  @Ignore("HADOOP-16845: Server side raw query consumption will lead to "
+      + "ignoring invalid continuation token.")
```

Comment:
Perhaps a better message would be, "HADOOP-16845: Invalid continuation tokens are ignored by the ADLS Gen2 service, so we are disabling this test until the service is fixed."